### ISSUE TITLE
Search widget inside calcite panel hides search suggestions

### ISSUE THREAD

**Issue by msamwill** [Esri/calcite-design-system#7609]
### Check existing issues

- [X] I have [checked for existing issues](https://github.com/Esri/calcite-design-system/issues) to avoid duplicates

### Actual Behavior

Search widget inside calcite panel hides search suggestions.  If you make CSS changes it looks better and you can see search suggestions but panel has excessive white space when opened and a fixed height is not ideal

#search-container.esri-search.esri-widget {
    width: 100% !important;
    align-items: flex-start;
    min-height: 360px;
    padding: 0px;
    overflow: auto;
}

### Expected Behavior

Expect panel to adjust when search suggestions are returned and to fit into panel better without CSS changes.  I expect this because most other widgets work this way.

### Reproduction Sample

https://codepen.io/msamwill/pen/BavobyW

### Reproduction Steps

1. Open search panel
2. Attempt to pick Congressional Districts in Search Picklist
![image](https://github.com/Esri/calcite-design-system/assets/7591429/16916118-37a4-4b0e-b972-b07fb58a66f5)
It should look like this:
![image](https://github.com/Esri/calcite-design-system/assets/7591429/5af06f29-732d-4fad-873c-c60058992dc0)
3. Type 370 in search box, you should see a list of 6 possible results, but the results will be hidden.
Looks like this:
![image](https://github.com/Esri/calcite-design-system/assets/7591429/eab96203-f533-4caf-bdf2-9d79a36492be)
Should look like this:
![image](https://github.com/Esri/calcite-design-system/assets/7591429/3e286ee9-cb9d-4447-a842-db3bb4915879)

### Reproduction Version

4.27

### Priority impact

p1 - need for current milestone

### Impact

Delaying replacement of ESRI JavaScript Api 3.x website.

### Calcite package

- [ ] @esri/calcite-components
- [ ] @esri/calcite-components-react

### Esri team

N/A

**Comment by msamwill** [comment]
CSS adjustments to the search widget help improve appearance [](https://codepen.io/msamwill/pen/wvRGvzg) looking for a way to make the height of the panel adjust dynamically to the search widget content.

**Comment by msamwill** [comment]
[Code Pen with CSS Adjustments](https://codepen.io/msamwill/pen/wvRGvzg)

**Comment by geospatialem** [comment]
This is out of scope of Calcite, and related to the structure of the widget in the Maps SDK for JavaScript. 

@msamwill For future consideration of Maps SDK for JavaScript issues, bugs and enhancements are filed via [Esri Support](https://support.esri.com/) for the team, so progress can be tracked. I opened an issue for the team with the above information and limited workaround mentioned in the issue above. Closing per the scope and new issue with the Maps SDK for JavaScript team.
